src/modules/file-input.marko

```diff
@@ -0,0 +1,98 @@
+<div id="file-input">
+    <section-header metadata=input.metadata name=input.name alpha="true"/>
+
+    <p>
+        The file-input lets the user choose one or more files from their device storage. Once chosen, the files can be uploaded to a server using${" "}
+        <a
+            target="_blank"
+            href="https://developer.mozilla.org/en-US/docs/Learn/Forms"
+        >
+            form submission
+        </a>
+         , or manipulated using JavaScript code and
+        <a
+            target="_blank"
+            href="https://developer.mozilla.org/en-US/docs/Web/API/File_API/Using_files_from_web_applications"
+        >
+            the File API
+        </a>
+         .
+    </p>
+    <p>
+        The entire element can function as a dropzone for drag and drop, however this functionality requires JavaScript and is outside the scope of a CSS library such as eBay Skin.
+    </p>
+
+    <h3>Base</h3>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="file-input">
+                <input
+                    id="file-input-1"
+                    class="file-input__input"
+                    type="file"
+                    multiple
+                >
+                <div class="file-input__container">
+                    <div class="file-input__upload-icon">
+                        <svg class="icon" aria-hidden="true">
+                            <use href="#icon-upload-24"/>
+                        </svg>
+                    </div>
+                    <div class="file-input__content">
+                        <span class="file-input__content-header">
```

Comment:
I missed this, but should we use a heading here? @saiponnada it would probably make sense, right? If so, the coreui component would need to use the `as=h2` as=`h3` etc.

Reply:
Although the input is positioned before the heading element in the DOM. If we make it a header the input would probably have to go after the heading.